test/libyul/yulInterpreterTests/side_effect_free.yul

```diff
@@ -6,7 +6,6 @@
 	pop(extcodehash(0))
 	pop(returndatasize())
 	pop(sload(0))
-	pop(pc())
```

Comment:
Why was this and the test above removed? It is deprecated but still has to produce proper code.

Reply:
It's a warning now but going to be removed in the next breaking release, which is about to be soon(?).